### PR TITLE
Feature/tag

### PR DESCRIPTION
**Pull request by sShayron**
- Add support to include tags in the sounds.

### CODE REVIEW

**Comment by sShayron** [comment]
image
image
image

image
image
image

File: src/App.tsx [hunk 1]
```diff
@@ -131,20 +145,6 @@ export function App() {
...
-            value={chatClient}
-            onChange={(event) =>
-              setChatClient(event.target.value as ChatClient)
-            }>
-            <option value="discord">Discord</option>
-            <option value="telegram">Telegram</option>
-            <option value="browser">Browser</option>
-          </Select>
-        </Box>
-
```
Comment (armand1m) [inline src/App.tsx]: fascism

File: src/App.tsx [hunk 2]
```diff
@@ -177,14 +178,15 @@ export function App() {
 
                 return (
                   <ButtonsSection>
-                    {filtered.map((sound) => (
+                    {filtered.map(({ item }: any) => (
```
Comment (armand1m) [inline src/App.tsx]: you definitely shouldn't need this

Comment (armand1m) [reply src/App.tsx]: please type the `filtered` array correctly and don't use any here

File: src/features/api/useMuminstApi.ts [hunk 3]
```diff
@@ -54,14 +56,16 @@ export const playSound = (chatClient: ChatClient, sound: Sound) => {
   });
 };
 
-export const uploadFiles = (
+export const uploadFiles = async (
   files: File[],
+  tags: string[] = [],
```
Comment (armand1m) [inline src/features/api/useMuminstApi.ts]:
```suggestion
  tags: string[],
```

File: src/App.tsx [hunk 4]
```diff
@@ -30,7 +29,8 @@ import {
   ChatClient,
   useMuminstApi,
 } from 'features/api/useMuminstApi';
-import { fuzzyFilter } from 'features/search/useFuzzyFilter/fuzzyFilter';
```
Comment (armand1m) [inline src/App.tsx:33]: delete the code and also remove `fuzzaldrin` from the dependency list

**Review by armand1m — CHANGES_REQUESTED** [review]
looking very nice!

File: src/App.tsx (hunk 1 above)
Comment (sShayron) [reply src/App.tsx]: ?

File: src/features/api/useMuminstApi.ts (hunk 3 above)
Comment (sShayron) [reply src/features/api/useMuminstApi.ts]: done

File: src/App.tsx (hunk 2 above)
Comment (sShayron) [reply src/App.tsx]: done

File: src/App.tsx (hunk 4 above)
Comment (sShayron) [reply src/App.tsx:33]: done

File: src/App.tsx (hunk 1 above)
Comment (armand1m) [reply src/App.tsx]: Screen Shot 2021-09-09 at 2 13 27 AM

**Review by armand1m — COMMENTED** [review]
looking good but can we have the dropdown back?

File: src/App.tsx (hunk 1 above)
Comment (sShayron) [reply src/App.tsx]: Sorry, I added back the client select

**Review by armand1m — APPROVED** [review]

**Pull request merged**